certbot/compat/security.py

```diff
@@ -0,0 +1,307 @@
+"""Compat module to handle files security on Windows and Linux"""
+from __future__ import absolute_import
+
+import os
+import stat
+
+try:
+    import pwd  # pylint: disable=import-error
+except ImportError:  # pragma: no cover
+    pwd = None  # type: ignore
```

Comment:
mypy has an optional type, does that not work here?

Reply:
I used the same approach that is done on several locations in the Certbot codebase, But indeed, it is smarter, I will try that.

Reply:
But it does not work... you cannot assign a type on an import statement. Then, any try to add a type at the second statement (like `pwd` = None) that would solve the issue with mypy will fail, and of course mypy will fail without `# type: ignore`.

Reply:
enough of a reason to ignore it for me